src/metabase/public_settings.clj

```diff
@@ -325,24 +325,7 @@
 (defsetting query-caching-max-ttl
   (deferred-tru "The absolute maximum time to keep any cached query results, in seconds.")
   :type    :double
-  :default (* 60.0 60.0 24.0 100.0) ; 100 days
-  :audit   :getter)
-
-;; TODO -- this isn't really a TTL at all. Consider renaming to something like `-min-duration`
-(defsetting query-caching-min-ttl
```

Comment:
meta question: I wonder whether we need to clean up settings if we remove it. there is a chance someone might reuse the setting name in the future and having an existing value could be unexpected.

Reply:
I plan to do that in 51, I don't want to drop values just yet and parsing from `cache_config` into older ways is going to be a nightmare.